addons/addon-base-raas/packages/base-raas-services/lib/aws-accounts/aws-cfn-service.js

```diff
@@ -236,16 +266,20 @@ class AwsCfnService extends Service {
 
     const awsAccountsService = await this.service('awsAccountsService');
     const accountsList = await awsAccountsService.list();
-
+    // const stackOutputs = ['externalId', 'vpcId', 'subnetId', 'encryptionKeyArn', 'cfnStackId', 'roleArn'];
     const newStatus = {};
     const errors = {};
     const idList = accountsList.forEach(account => account.accountId);
     let res;
     let errorMsg = '';
 
     const checkPermissions = async account => {
-      if (account.cfnStackName === '') {
-        res = 'NEEDSONBOARD';
+      errorMsg = '';
+      if (
+        account.permissionStatus === 'NEEDSONBOARD' ||
+        account.permissionStatus === 'PENDING' // Backend workflow will bring the account out of PENDING
```

Comment:
which one? This comment is still current, I just wanted to indicate that accounts in PENDING state are skipped because they're being handled by another resource

Reply:
Ah, I thought you forgot to update this one. Backend workflow in the context of SWB would be interpreted as an actual workflow, since we're not using a workflow here, it's clearer to say Backend Lambda <lambda name> will bring the account out of PENDING.

Reply:
ah okay, i'll make that fix